libc/test/src/__support/str_to_float_comparison_test.cpp

```diff
@@ -8,8 +8,6 @@
 
 // #include "src/__support/str_float_conv_utils.h"
 
-#include <stdlib.h>
```

Comment:
This is an unusual test (that I honestly need to refactor and maybe remove) but for now this include needs to stay.

Reply:
sounds good. thanks

Reply:
@michaelrj-google hey, i dont see any direct usage of this `#include <stdlib.h>` header file here. ive ran both the overlay and full build mode and the build and tests for both modes are successful

Reply:
This test is built differently from the rest of them since it's mostly an ugly hack I used for performance/correctness testing while implementing strtofloat. If you want to run it then you need to run `ninja libc_str_to_float_comparison_test`. It needs the `stdlib` header since it's using the public names of the string to float functions, which is because I wanted to be able to switch it from using my implementation and the system implementation just in cmake.

Reply:
ok sounds good. thanks. well i have just addressed Nicks last review.